angular.json

```diff
@@ -4,25 +4,35 @@
   "newProjectRoot": "projects",
   "projects": {
     "metis": {
-      "root": "",
-      "sourceRoot": "src",
+      "root": "projects/metis",
```

Comment:
Now that you're setting this root, should you not define all paths below relative to this root?

Reply:
Those were my thoughts at first, but it only works when configured as it is here.

Reply:
Odd... but I believe you :)